### PR TITLE
[docs] add YCQL example

### PR DESCRIPTION
**Pull request by ddhodge**
add a basic YCQL example

@netlify /latest/yugabyte-cloud/cloud-basics/create-databases/

### CODE REVIEW

**Review by VijiYB — APPROVED** [review]
Looks good to me. Thanks.

File: docs/content/latest/yugabyte-cloud/cloud-basics/create-databases-ycql.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+ <li >
+    <a href="../create-databases-ycql/" class="nav-link active">
+      <i class="icon-cassandra" aria-hidden="true"></i>
+      YCQL
+    </a>
+  </li>
+  
+</ul>
+
+Using the `ycqlsh` shell, you can interact with your YugabyteDB database using the YCQL API. The following exercise shows how you can use ycqlsh to create a keyspace, add a table, insert some data, and run a simple query.
```
Comment (polarweasel) [inline docs/content/latest/yugabyte-cloud/cloud-basics/create-databases-ycql.md:37]:
```suggestion
Using the `ycqlsh` shell, you can interact with your YugabyteDB database using the YCQL API. In the following exercise, you'll use ycqlsh to create a keyspace, add a table, insert some data, and run a simple query.
```

File: docs/content/latest/yugabyte-cloud/cloud-basics/create-databases-ycql.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+1. Create a keyspace called 'myapp'.
+
+    ```sql
+    ycqlsh> CREATE KEYSPACE myapp;
+    ```
+
+1. Create a table named `stock_market`, which can store stock prices at various timestamps for different stock ticker symbols.
+
+    ```sql
```
Comment (polarweasel) [inline docs/content/latest/yugabyte-cloud/cloud-basics/create-databases-ycql.md:53]: FYI, there's a `cql` option too. Highlighting may differ slightly and theoretically be better.
````suggestion
    ```cql
````

File: docs/content/latest/yugabyte-cloud/cloud-basics/create-databases-ycql.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+```sql
+ycqlsh> INSERT INTO myapp.stock_market (stock_symbol,ts,current_price) VALUES ('FB','2017-10-26 09:00:00',170.63);
+INSERT INTO myapp.stock_market (stock_symbol,ts,current_price) VALUES ('FB','2017-10-26 10:00:00',170.1);
+```
+
+```sql
+ycqlsh> INSERT INTO myapp.stock_market (stock_symbol,ts,current_price) VALUES ('GOOG','2017-10-26 09:00:00',972.56);
+INSERT INTO myapp.stock_market (stock_symbol,ts,current_price) VALUES ('GOOG','2017-10-26 10:00:00',971.91);
+```
```
Comment (polarweasel) [inline docs/content/latest/yugabyte-cloud/cloud-basics/create-databases-ycql.md:79]: You could combine these... (And drop the prompts 🤔)
````suggestion
```cql
INSERT INTO myapp.stock_market (stock_symbol,ts,current_price) VALUES ('AAPL','2017-10-26 09:00:00',157.41);
INSERT INTO myapp.stock_market (stock_symbol,ts,current_price) VALUES ('AAPL','2017-10-26 10:00:00',157);

INSERT INTO myapp.stock_market (stock_symbol,ts,current_price) VALUES ('FB','2017-10-26 09:00:00',170.63);
INSERT INTO myapp.stock_market (stock_symbol,ts,current_price) VALUES ('FB','2017-10-26 10:00:00',170.1);

INSERT INTO myapp.stock_market (stock_symbol,ts,current_price) VALUES ('GOOG','2017-10-26 09:00:00',972.56);
INSERT INTO myapp.stock_market (stock_symbol,ts,current_price) VALUES ('GOOG','2017-10-26 10:00:00',971.91);
```
````

File: docs/content/latest/yugabyte-cloud/cloud-basics/create-databases-ycql.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+  
+</ul>
+
+Using the `ycqlsh` shell, you can interact with your YugabyteDB database using the YCQL API. The following exercise shows how you can use ycqlsh to create a keyspace, add a table, insert some data, and run a simple query.
+
+## Create a keyspace and add a table
+
+To create a keyspace and add a table, do the following:
+
+1. Connect to your cluster using `ycqlsh` using the [Client Shell](../connect-to-clusters#connect-via-client-shell) from your computer.
```
Comment (polarweasel) [inline docs/content/latest/yugabyte-cloud/cloud-basics/create-databases-ycql.md:43]: Do these instructions apply to the cloud shell, as well? (If so, worth mentioning, as that's easier.)

File: docs/content/latest/yugabyte-cloud/cloud-basics/create-databases-ycql.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+
+1. Create a keyspace called 'myapp'.
+
+    ```sql
+    ycqlsh> CREATE KEYSPACE myapp;
```
Comment (polarweasel) [inline docs/content/latest/yugabyte-cloud/cloud-basics/create-databases-ycql.md:48]: I am tempted to introduce a style convention whereby we include the prompt once or twice, and then omit for the rest of a given procedure/page. I know they don't get copied when you use the button, but they still are kind of annoying. Style guide discussion fodder.

File: docs/content/latest/yugabyte-cloud/cloud-basics/create-databases.md [hunk 6]
```diff
@@ -1,22 +1,45 @@
 ---
```
Comment (polarweasel) [inline docs/content/latest/yugabyte-cloud/cloud-basics/create-databases.md:1]: Same comments as the other file apply to this one...

File: docs/content/latest/yugabyte-cloud/cloud-basics/create-databases.md [hunk 7]
```diff
@@ -1,22 +1,45 @@
...
+      <i class="icon-cassandra" aria-hidden="true"></i>
+      YCQL
+    </a>
+  </li>
+  
+</ul>
+
+Using the `ysqlsh` shell, you can interact with your YugabyteDB database using the YSQL API. The following exercise shows how you can use ysqlsh to create a database, load a sample dataset, and run a simple query.
+
+Sample datasets that you can use to test out YugabyteDB are located in the `share` directory of your YugabyteDB Client Shell installation. The datasets are provided in the form of SQL script files. (The datasets are also available in the [sample directory of the YugabyteDB GitHub repository](https://github.com/yugabyte/yugabyte-db/tree/master/sample)). This exercise uses the [Retail Analytics](../../../develop/realworld-apps/retail-analytics/) dataset.
```
Comment (polarweasel) [inline docs/content/latest/yugabyte-cloud/cloud-basics/create-databases.md:42]:
```suggestion
Your YugabyteDB client shell installation includes sample datasets you can use to test out YugabyteDB. These are located in the `share` directory. The datasets are provided in the form of SQL script files. (The datasets are also available in the [sample directory of the YugabyteDB GitHub repository](https://github.com/yugabyte/yugabyte-db/tree/master/sample)). This exercise uses the [Retail Analytics](../../../develop/realworld-apps/retail-analytics/) dataset.
```

**Review by polarweasel — APPROVED** [review]
Preemptive approval with suggestions and comments. Merge when you're happy :)

**Pull request merged**